src/main/java/seedu/duke/exercise/ExerciseList.java

```diff
@@ -0,0 +1,76 @@
+package seedu.duke.exercise;
+
+import java.util.ArrayList;
+
+public class ExerciseList {
+
+    public static final String MESSAGE_EXERCISE_ADDED = "An exercise has been added:";
+    private static final String MESSAGE_EXERCISE_DELETED = "You have removed the exercise:";
+    private static final String MESSAGE_EMPTY_EXERCISE_LIST = "No exercise is found.";
+    private ArrayList<Exercise> exerciseList = new ArrayList<>();
+
+    /**
+     * Returns exercise item at a specific index in the exercise list.
+     *
+     * @param index Index of the exercise item.
+     * @return exercise item in the exercise list.
+     */
+    public Exercise getExercise(int index) {
+        return this.exerciseList.get(index);
+    }
+
+    /**
+     * Returns the number of exercises in the exercise list.
+     *
+     * @return Number of exercises in the exercise list.
+     */
+    public int getSize() {
```

Comment:
We can discuss and standardize this method name, whether to just call it `size()` or `getSize()` (in the food list class it is called `size()`.

Reply:
Okay! The reason why I use naming like `size()`, `get()` and `add()` in food list is because I want to follow the naming in array list, but ya it is better to follow the coding standards :)